### ISSUE TITLE
Plugin doesn't load on TypeDoc v0.22.6

### ISSUE THREAD

**Issue by f-t-alves** [tgreyuk/typedoc-plugin-inline-sources#1]
Currently throwing error:

```
Error: Error [ERR_PACKAGE_PATH_NOT_EXPORTED]: Package subpath './dist/lib/converter/components' is not defined by "exports" in OBFUSCATED_PROJECT_PATH\node_modules\typedoc\package.json
    at throwExportsNotFound (internal/modules/esm/resolve.js:285:9)
    at packageExportsResolve (internal/modules/esm/resolve.js:508:3)
    at resolveExports (internal/modules/cjs/loader.js:450:36)
    at Function.Module._findPath (internal/modules/cjs/loader.js:490:31)
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:888:27)
    at Function.Module._load (internal/modules/cjs/loader.js:746:27)
    at Module.require (internal/modules/cjs/loader.js:974:19)
    at require (internal/modules/cjs/helpers.js:92:18)
    at Object.<anonymous> (OBFUSCATED_PROJECT_PATH\node_modules\typedoc-plugin-inline-sources\dist\plugin.js:11:22)
    at Module._compile (internal/modules/cjs/loader.js:1085:14)
```

Might be related to [this issue](https://github.com/TypeStrong/TypeDoc/issues/1696). In the same issue, maintainer mentions usage of `Converter` and `ConvertedComponent` by plugins will be deprecated, so probably something else to be looking out for.

**Comment by tgreyuk** [comment]
Sorry completely missed this (I actually didn't realise anyone was using the plugin :). fixed in 1.0.1.

**Comment by f-t-alves** [comment]
Awesome! Thanks, mate. Definitely plan on using it now!
